web/vibe/web/web.d

```diff
@@ -378,6 +379,43 @@ void redirect(string url)
 	ctx.res.redirect(fullurl);
 }
 
+/**
+	Sets a response header.
+
+	Params:
+		name = name of the header to set
+		value = value of the header to set
+
+	Note that this may only be called from a function/method
+	registered using registerWebInterface.
+*/
+void header(string name, string value)
+{
+	assert(s_requestContext.req !is null, "header() used outside of a web interface request!");
+	alias ctx = s_requestContext;
+	ctx.res.headers[name] = value;
+}
+
+/**
+	Sets the response status code.
+
+	Params:
+		statusCode = the HTTPStatus code to send to the client
+
+	Note that this may only be called from a function/method
+	registered using registerWebInterface.
+*/
+@property void status(int statusCode)
```

Comment:
I'm not *quite* sure I'd agree there 😄 ...but that would actually be UFCS, not `@property`.